packages/grafana-ui/src/types/datasource.ts

```diff
@@ -84,6 +84,11 @@ export interface DataSourcePluginMeta extends PluginMeta {
   category?: string;
   queryOptions?: PluginMetaQueryOptions;
   sort?: number;
+
+  /**
+   * By default, hidden queries are not passed to the datasource
+   */
+  processHiddenQueries?: boolean;
```

Comment:
always find verb / action named properties that are bool flags wrong but was hard pressed come up with an alterantive here. 

* needsHiddenQueries
* handlesHiddenQueries
* ?

Reply:
maybe supportsHiddenQueries?